lib/mongorito.js

```diff
@@ -25,6 +16,14 @@ const is = require('is_js');
 
 const Query = require('./query');
 
+const MongoClient = mongodb.MongoClient;
+
+class EnforcerError extends Error {
```

Comment:
Hrm, looks like I was having issues early on in my refactor for the driver and mistook those for singleton issues. I've changed back. see latest commits.